tfjs-backend-webgpu/src/kernels/binary_op_shared_webgpu.ts

```diff
@@ -53,14 +54,14 @@ export class BinaryOpSharedProgram implements WebGPUProgram {
     this.dispatch = computeDispatch(
         this.dispatchLayout, this.outputShape, this.workGroupSize,
         [this.workPerThread, 1, 1]);
-    this.shaderKey = `binaryShared_${op}`;
+    this.shaderKey = `binaryShared_${op}_${this.lastDimensionSize}`;
```

Comment:
Still missing `sizeFit`?

Reply:
Thanks Jiajia, you are right, fit is needed here (And matmul/conv2d).  However this is not introduced by this PR and the goal of this PR is to uniformilize shape, and the change is not trivial, I prefer to fix this in a followup PR.

Reply:
Hi Xing, in fact, it's introduced by this PR. Previously, the inputShape/OutputShape is as part of shader key. So you don't need to consider the fit issue. However, in this patch, you removed the shapes form shader key. Then you must add the `fit` to the shader key. Otherwise, your change will bring potential bugs.

Reply:
Oh, yes,  I will update it.